### ISSUE TITLE
Several questions on the MOT config settings

### ISSUE THREAD

**Issue by shwoo93** [open-mmlab/mmtracking#130]
Hello, 

I am new to MOT.
I have several questions below:

1. If my understanding is correct, the public stands for the detection boxes provided by the challenge to ensure the comparability between different tracking methods (so we use the provided bounding boxes for every video), whereas the private stands for detection boxes from any user-trained object detector (so we use the generated bounding boxes). Is my understanding correct?

2. I see the MOT17 'train' dataset is halved for train and test. Is this common in MOT?

3. why the config file of tracktor_faster-rcnn_r50_fpn_4e_mot17-private.py sets the test_set argument as 'train' instead of 'test'?

4. what is the purpose of the public-half_search config file? Is it to set the proper hyperparameters? If then, what hyperparameter we especially target for searching? (e.g., obj_score_thr)

**Comment by OceanPang** [comment]
1. Yes. You are correct. 
2. This is a practice following [CenterTrack](https://arxiv.org/abs/2004.01177), because the official dataset does not provide the validation set. 
3. Because generally the performance on the test set must be uploaded to the website. Normally we can use `train` for a performance checking.
4. Please refer to the [README](https://github.com/open-mmlab/mmtracking/blob/master/docs/useful_tools.md)

**Comment by shwoo93** [comment]
Thanks for your kind reply.
